webhooks/webhooks.helper.ts

```diff
@@ -0,0 +1,300 @@
+import { StepLogger } from '../../../../core/logger/step-logger';
+import { ExpectationHelper } from '../../../components/misc-utils/expectation-helper';
+import { CommonPageHelper } from '../common/common-page.helper';
+import { PageHelper } from '../../../components/html/page-helper';
+import { TextBoxHelper } from '../../../components/html/textbox-helper';
+import { ElementHelper } from '../../../components/html/element-helper';
+import { ManuscriptLoginPageHelper } from '../manuscript-app/manuscript-login/login-page.helper';
+import { ManuscriptHomePageHelper } from '../manuscript-app/home-page/home-page.helper';
+import { WaitHelper } from '../../../components/html/wait-helper';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400120)
```suggestion
import { WaitHelper } from "../../../components/html/wait-helper";
```